### PR TITLE
THIS IS DAYLIGHT ROBBERY

### PR DESCRIPTION
**Pull request by jspspike**

### CODE REVIEW

**Comment by rrbutani** [comment]
Please don't deface our repositories.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# pacbot Hardware Access Layer
+
+## Features:
+- [x] Drives DC motors over an H-Bridge with PWM
+- [x] Reads high frequency quadrature signals
+- [ ] Configures and reads from multiple VL6180X distance sensors
+- [x] Uses the super secret DMP on the MPU6050
```
Comment (rrbutani) [inline README.md:7]: This is untrue

Comment (jspspike) [reply README.md:7]: yeah the uses the super secret dmp was not checked

Comment (rrbutani) [reply README.md:7]: no, I'm saying we _don't_ use the DMP

Comment (jspspike) [reply README.md:7]: but in that branch it does though, or at least there was a driver for it

**Pull request closed without merging**